lib/phoenix_html/tag.ex

```diff
@@ -201,58 +201,28 @@ defmodule Phoenix.HTML.Tag do
         {[~s'<input name="#{@csrf_param}" type="hidden" value="', csrf_token, ~s'">'], opts}
 
       {true, opts} ->
-        csrf_token = csrf_token_value(to)
+        csrf_token = Phoenix.HTML.csrf_token_value(to)
         {[~s'<input name="#{@csrf_param}" type="hidden" value="', csrf_token, ~s'">'], opts}
 
       {false, opts} ->
         {[], opts}
     end
   end
 
-  @doc """
-  Returns the `csrf_token` value to be used by forms, meta tags, etc.
-
-  By default, CSRF tokens are generated through `Plug.CSRFProtection`
-  which is capable of generating a separate token per host. Therefore
-  it is recommended to pass the `URI` of the destination as argument.
-  If none is given `%URI{host: nil}` is used, which implies a local
-  request is being done.
-  """
-  def csrf_token_value(to \\ %URI{host: nil}) do
-    {mod, fun, args} = Application.fetch_env!(:phoenix_html, :csrf_token_reader)
-    apply(mod, fun, [to | args])
-  end
+  @doc false
```

Comment:
Keeping the same pattern as used on https://github.com/phoenixframework/phoenix_html/blob/415402ceabf67c26a7f7ee91f1a513c5860a2392/lib/phoenix_html/tag.ex#L113-L114